.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -17,6 +17,14 @@ body:
       placeholder: e.g. v9.0.0
     validations:
       required: true
+  - type: input
+    id: development_environment
+    attributes:
+      label: Development environment
+      description: Tell us what development environment you use
+      placeholder: e.g. Visual Studio
+    validations:
+      required: true
```

Comment:
```suggestion
    id: platform
    attributes:
      label: Platform
      description: Tell us platform and environment you use
      placeholder: e.g. Visual Studio simulator, or STM32F769 Discovery with Zephyr, or Eclipse with SDL on Linux 
    validations:
      required: true
```

Reply:
Yes, the word platform does make more sense.